src/data/reducers/auth.js

```diff
@@ -46,7 +47,7 @@ export default function (state = initialState, action) {
         firstName: null,
         lastName: null, 
         email: null,
-        id: null,
+        userId: null,
```

Comment:
should we add `teamId: null` here (and below)?

Reply:
Good catch, I just modified the reducers to set everything (including `teamId`) to null if the user isn't considered logged in